inbm/cloudadapter-agent/cloudadapter/cloud/client/inbs_cloud_client.py

```diff
@@ -112,6 +114,53 @@ def publish_telemetry(self, key: str, value: str, time: datetime) -> None:
 
         pass  # INBS is not yet ready to receive telemetry
 
+    def publish_update(self, key: str, value: str) -> None:
+        """Publishes an update to the cloud
+
+        @param message: node update message to publish
+        @exception PublishError: If publish fails
+        """
+        if self._grpc_channel is None:
+            raise PublishError("gRPC channel not set up before calling InbsCloudClient.publish_update")            
+    
+        # Turn the message into a dict
```

Comment:
Our standard for communication between agents is XML so we have a schema and validation (similar to how the commands from UDM come in as gRPC and get converted to XML in cloudadapter)